reconcile_test.go

```diff
@@ -318,7 +318,7 @@ func TestReconcile(t *testing.T) {
 					// already a non-voter so nothing should be done
 					"0a79bbf7-7113-4947-a257-6179326f188c",
 				},
-				// no leader ship transfer should be done because we did demotions
+				// should transfer leadership because no demotions actually needed performing
```

Comment:
Fixing an inaccurate comment, originally I had copy/pasted this test with a few modifications and didn't update the comment.